_episodes/01-numpy.md

```diff
@@ -57,7 +57,7 @@ weight_kg = 60
 {: .language-python}
 
 From now on, whenever we use `weight_kg`, Python will substitute the value we assigned to
-it. In essence, **a variable is just a name for a value**.
+it. In essence, **a variable is a name for a value**.
```

Comment:
Perhaps better to call it a "place holder for one of a set of possible values" or something along those lines. Given the audience is primarily scientists, I think they get what a variable is.  Maybe even drop the whole "In essence..." sentence.

Reply:
> Perhaps better to call it a "place holder for one of a set of possible values"

I don't think this phrasing is better as it raises such questions as "what set of possible values?", what is a set?, which possible values?, does placeholder mean something in Python?. Current explanation uses terminology that is well known ("name", "value") and that has been just discussed ("variable").

At this point, I would prefer to keep changes to a minimum.

>  Maybe even drop the whole "In essence..." sentence.

This sentence is the culmination of a very brief introduction to variables. It is important for those who do not know what a variable is before the workshop. I'd like to emphasize that our lesson is titled "**Introduction to** Programming with Python" and as recently as 5/29/2019 I taught this lesson and there were learners who did not know anything about programming. I would not say that this is typical, but there is nothing wrong with that and even this brief introduction is better than nothing. I am considering to propose to split this brief introduction into a separate episode but this would have to be done in a separate PR and we would have to discuss it with those who teach this lesson.

Reply:
How about
> Essentially, a variable is a name for a value.

Reply:
additional commit(s) coming shortly